cmd/get.go

```diff
@@ -81,14 +87,45 @@ and provides a fast and easy alternative to a package manager.`,
 
 		outFilePath := path.Join(tmp, fileName)
 
-		out, err := os.Create(outFilePath)
-		if err != nil {
-			return err
-		}
-		defer out.Close()
+		if tool.IsArchive() {
+			outFilePathDir := filepath.Dir(outFilePath)
+			outFilePath = path.Join(outFilePathDir, tool.Name)
+			if strings.Contains(strings.ToLower(operatingSystem), "mingw") && tool.NoExtension == false {
+				outFilePath += ".exe"
+			}
+			r := ioutil.NopCloser(res.Body)
+			if strings.HasSuffix(downloadURL, "tar.gz") {
+				untarErr := helm.Untar(r, outFilePathDir)
+				if untarErr != nil {
+					return untarErr
+				}
+			} else if strings.HasSuffix(downloadURL, "zip") {
+				buff := bytes.NewBuffer([]byte{})
+				size, err := io.Copy(buff, res.Body)
+				if err != nil {
+					return err
+				}
+				reader := bytes.NewReader(buff.Bytes())
+				zipReader, err := zip.NewReader(reader, size)
```

Comment:
I think we can improve the API here - for instance, let's pass in the bytes directly to the method and have it create the zip reader via the zip.NewReader call.